### DISCUSSION TITLE
Google Social Icon rejected by Google

### DISCUSSION THREAD

**Discussion by meezaan** [keycloak/keycloak#24220] [Q&A]
Hi Folks,

I've configured social login with several different providers.

Google now verifies how the Login with Google button appears and they're simply rejecting the icon.

Has anyone else run into this issue? If so, do you know how best to deal with this rather than sticking an ugly if statement in https://github.com/keycloak/keycloak/blob/main/themes/src/main/resources/theme/base/login/login.ftl#L96?

Thanks.

**Comment by anowak-ct** [comment]
I have the exact same issue. Out app, which is using Google as ID provider for login, has been rejected by Google's app review - with the exact same reason.
See 
https://github.com/keycloak/keycloak/issues/24298

**Reply by meezaan** [reply]
> I have the exact same issue. Out app, which is using Google as ID provider for login, has been rejected by Google's app review - with the exact same reason.
> 
> See 
> 
> https://github.com/keycloak/keycloak/issues/24298
> 
> 
> 
> 

I have switched to https://github.com/lukin/keywind as the theme which uses provider branding for all the  providers and have resubmitted the request to Google (although there is no response yet).
